databend-jdbc/src/main/java/com/databend/jdbc/DatabendDriverUri.java

```diff
@@ -57,17 +58,22 @@ public final class DatabendDriverUri {
 
     private DatabendDriverUri(String url, Properties driverProperties)
             throws SQLException {
-        Map.Entry<URI, Map<String, String>> uriAndProperties = parse(url);
-        this.properties = mergeProperties(uriAndProperties.getKey(), uriAndProperties.getValue(), driverProperties);
+        Map.Entry<DatabendNodes, Map<String, String>> uriAndProperties = parse(url);
```

Comment:
why it's a `Map.Entry` ? can i understand it as a tuple `(DatabendNodes, Map<String,String>)` in rust?